editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -4419,6 +4431,47 @@ bool Node3DEditorViewport::_create_instance(Node *parent, String &path, const Po
 	return true;
 }
 
+bool Node3DEditorViewport::_create_audio_node(Node *parent, String &path, const Point2 &p_point) {
+	Ref<Resource> res = ResourceLoader::load(path);
```

Comment:
```suggestion
	Ref<AudioStream> audio = ResourceLoader::load(path);
```
You can assign it directly here. It will be null if type does not match.